stingray/gti.py

```diff
@@ -116,10 +116,9 @@ def create_gti_mask(time, gtis, safe_interval=0, min_length=0,
 
     check_gtis(gtis)
 
-    dt = assign_value_if_none(dt,
-                              np.zeros_like(time) +
-                              np.median(np.diff(time)) / 2)
+    dt_array = np.diff(np.sort(time))
```

Comment:
ah, that was me not paying attention because I copied some stuff from `lightcurve.py`, where `dt_array` is used several times. Should be fixed now?